ui/cypress/e2e/explorer.test.ts

```diff
@@ -359,16 +358,16 @@ describe('DataExplorer', () => {
       cy.getByTestID('time-machine-submit-button').should('be.disabled')
 
       cy.getByTestID('flux-editor').within(() => {
-        cy.get('textarea').type('yo', {force: true})
+        cy.get('.CodeMirror').type('yo')
         cy.getByTestID('time-machine-submit-button').should('not.be.disabled')
       })
     })
 
     it('disables submit when a query is deleted', () => {
```

Comment:
```suggestion
for (let i = 0; i < 50; i++) {
    it('disables submit when a query is deleted', () => {
```

Reply:
@hoorayimhelping here's an example of how to loop one test.

Reply:
sweet, thanks